src/libs/ajax/Metrics.ts

```diff
@@ -23,8 +23,7 @@ export const Metrics = (signal?: AbortSignal) => {
       event,
       properties: {
         ...details,
-        // Observation: The `true` and `false` sides of this ternary look like they may be reversed?  Perhaps someone who knows why it was done this way can comment explaining the logic here
-        distinct_id: isRegistered ? undefined : authStore.get().anonymousId,
```

Comment:
Looks like a newline got accidentally deleted here.

Reply:
Yes that was giving us a headache, thanks for the callout!